### PR TITLE
Update tests for libpostal integration

### PR DESCRIPTION
**Pull request by trescube**

### CODE REVIEW

**Comment by dianashk** [comment]
Thanks for updating these. We usually open an issue for each new regression and add a url to that issue in the test when marking it failing. This helps for us to not forget about the issues.

**Comment by trescube** [comment]
Issues added to pelias/pelias

File: test_cases/labels.json [hunk 1]
```diff
@@ -70,11 +70,12 @@
     },
     {
       "id": 6,
-      "status": "pass",
+      "status": "fail",
       "issue": "https://github.com/pelias/pelias/issues/294",
```
Comment (dianashk) [inline test_cases/labels.json]: this issue has been closed so a new one needs to opened and referenced here. just change issue to be an array of links.

File: test_cases/exact_matches.json [hunk 2]
```diff
@@ -97,9 +97,10 @@
     },
     {
       "id": 8,
-      "status": "pass",
+      "status": "fail",
       "user": "sevko",
```
Comment (dianashk) [inline test_cases/exact_matches.json]: add issue property to test and link to https://github.com/pelias/pelias/issues/427

File: test_cases/encoding.json [hunk 3]
```diff
@@ -45,10 +45,13 @@
     },
     {
       "id": 3,
-      "status": "pass",
+      "status": "fail",
       "endpoint": "search",
       "issue": "https://github.com/pelias/api/issues/600",
```
Comment (dianashk) [inline test_cases/encoding.json]: this issue has been closed once the test started passing. please add the new issue pelias/pelias#426 to the list. just change `issue` to be an array of links.

File: test_cases/admin_lookup.json [hunk 4]
```diff
@@ -494,10 +494,13 @@
     },
     {
       "id": "20",
-      "status": "pass",
+      "status": "fail",
```
Comment (dianashk) [inline test_cases/admin_lookup.json]: add issue https://github.com/pelias/pelias/issues/425

File: test_cases/address_matching.json [hunk 5]
```diff
@@ -5,7 +5,7 @@
   "tests": [
     {
       "id": 1,
-      "status": "pass",
+      "status": "fail",
```
Comment (dianashk) [inline test_cases/address_matching.json]: add issue property to test and reference https://github.com/pelias/pelias/issues/424

**Pull request merged**